helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/TestWagedRebalancerMetrics.java

```diff
@@ -123,6 +123,8 @@ public void testWagedRebalanceMetrics()
     // Cluster config change will trigger baseline recalculation and partial rebalance.
     when(clusterData.getRefreshedChangeTypes())
         .thenReturn(Collections.singleton(HelixConstants.ChangeType.CLUSTER_CONFIG));
+    // Update the config so the cluster config will be marked as changed.
```

Comment:
Add a field to the cluster config so the cluster config will be marked as changed in the change detector.